### PR TITLE
chore(vertexai): clean up chat example

### PR DESCRIPTION
**Pull request by jba**
## Description

Change the chat example to use more idiomatic Go.

## Checklist
- [x] I have followed [Contributing Guidelines from CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md)
- [x] **Tests** pass:   `go test -v ./..` (see [Testing](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#testing))
- [x] **Code formatted**:   `gofmt` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [x] **Vetting** pass:   `go vet` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: vertexai/chat/chat_test.go [hunk 1]
```diff
@@ -15,15 +15,19 @@
...
+	"context"
+	"os"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
 )
 
 func Test_makeChatRequests(t *testing.T) {
-	t.Skip("TODO(muncus): remove skip")
+	if os.Getenv("GOLANG_SAMPLES_PROJECT_ID") == "" {
```
Comment (muncus) [inline vertexai/chat/chat_test.go:26]: This check is already performed in `testutil.SystemTest` below, and is redundant here. please remove.

**Review by muncus — APPROVED** [review]
aside from the minor fix noted, this looks great. Thanks!

**Review by eliben — APPROVED** [review]

File: vertexai/chat/chat_test.go (hunk 1 above)
Comment (jba) [reply vertexai/chat/chat_test.go]: Done.

**Comment by jba** [comment]
@muncus suggestions?

**Pull request merged**